python/ray/runtime_context.py

```diff
@@ -160,6 +161,18 @@ def should_capture_child_tasks_in_placement_group(self):
         """
         return self.worker.should_capture_child_tasks_in_placement_group
 
+    @property
+    def assigned_resources(self):
```

Comment:
is this by default stable api instead of beta?

Reply:
guess so? think we only tag top level class right?